Source/FolioReaderContainer.swift

```diff
@@ -216,6 +216,23 @@ open class FolioReaderContainer: UIViewController {
 }
 
 extension FolioReaderContainer {
+    func alert(error: Error) {
+        let message = error.localizedDescription
+        let alertController = UIAlertController(
+            title: "Error",
```

Comment:
Could you add the title to the localized config? https://github.com/FolioReader/FolioReaderKit/blob/master/Source/FolioReaderConfig.swift#L203